### PR TITLE
List_switches

### PR DESCRIPTION
**Pull request by ron-unrau**
Trying the list_switches pull request again. See PR #569

### CODE REVIEW

**Comment by SahilTikale** [comment]
Ran the code, populated it with switches and tested it.
+1

**Comment by knikolla** [comment]
+1, Looks good.

**Comment by henn** [comment]
Is there a reason we didn't merge this yet?

**Comment by SahilTikale** [comment]
@knikolla mentioned that there was some dependency on the patch #596 that you and Ian are working on.
If you have no dependency I will go ahead and merge it.
Let me know.

File: haas/api.py [hunk 1]
```diff
@@ -687,6 +687,18 @@ def switch_delete_port(switch, port):
     db.session.delete(port)
     db.session.commit()
 
+@rest_call('GET', '/switches')
+def list_switches():
```
Comment (henn) [inline haas/api.py]: Hey Ron.

Could you add a `Schema` argument, similar to this:
```python
@rest_call('GET', '/projects', Schema({}))
def list_projects():
```

Now that we've merged #596, all calls should have a `Schema`.

Thx!

**Comment by SahilTikale** [comment]
@ron-unrau  once you add it, I will have a look at it and merge this PR.

**Comment by ron-unrau** [comment]
The 'git push' failed. It says I need to re-sync my branch with the master now that the repository has changed. I think it is suicide for me to do anything more.

**Comment by SahilTikale** [comment]
@henn I think we should merge this and fix it making a new PR. 
Let me know if you agree. I will merge it and make the necessary fix.

**Pull request merged**